torch/csrc/jit/codegen/cuda/glfdc/eval.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
+
+// Expression evaluator
+
+#include "bitvector.h"
+#include "expr.h"
+#include "sexpr.h"
+#include "sparse_map.h"
+#include "stack.h"
+
+#include <algorithm>
+#include <functional>
+
+#include "c10/util/Optional.h"
+
+namespace glfdc {
```

Comment:
Please place this name space inside our umbrella namespace, `torch::jit::fuser::cuda`.

Reply:
Will do

Reply:
All of glfdc is moved to `torch::jit::fuser::cuda` updated version